optimum/habana/transformers/models/mixtral/modeling_mixtral.py

```diff
@@ -294,10 +289,14 @@ def gaudi_mixtral_block_sparse_moe_forward(self, hidden_states: torch.Tensor) ->
     # router_logits: (batch * sequence_length, n_experts)
     router_logits = self.gate(hidden_states)
 
-    if dist and dist.is_initialized():
+    if is_deepspeed_available():
+        from deepspeed import comm as dist
+
         output_tensors = [router_logits.clone() for _ in range(dist.get_world_size())]
         dist.all_gather(output_tensors, router_logits)
         router_logits = torch.cat(output_tensors, dim=1)
+    else:
+        print("Not using HPU DeepSpeed.")
```

Comment:
Let's remove that actually, there will already be a similar log before if DeepSpeed is used.